collection-scripts/gather_sno

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NODES_PATH=${OUT:-"${BASE_COLLECTION_PATH}/nodes"}
+
+function get_system_data_off_sno {
+
+    #Create a path for each node
+    mkdir -p "$NODES_PATH"/"$1"
+    
+    local DEBUG_POD_NAME_PREFIX=""
+
+    #Start Debug pod force it to stay up until removed in "default" namespace
```

Comment:
We should not start a debug pod in the default namespace; must-gather already runs in its own namespace (and has admin privileges).

Reply:
I was following this template when writing my rule:
https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_core_dumps
Could you please guide me then how to do this properly?